include/cppkafka/topic_partition_list.h

```diff
@@ -37,6 +37,7 @@
 #include <set>
 #include <librdkafka/rdkafka.h>
 #include "macros.h"
+#include "metadata.h"
```

Comment:
I would just use a forward declaration here, this pulls in a bunch of crap that I tried avoided pulling in other headers (e.g. `kafka_handle_base.h`)

Reply:
ok will do